lib/fluent/plugin/out_redis_store.rb

```diff
@@ -11,18 +11,19 @@ class RedisStoreOutput < BufferedOutput
     config_param :timeout,   :float,   :default => 5.0
 
     # redis command and parameters
-    config_param :format_type,  :string,  :default => 'json'
-    config_param :store_type,   :string,  :default => 'zset'
-    config_param :key_prefix,   :string,  :default => ''
-    config_param :key_suffix,   :string,  :default => ''
-    config_param :key,          :string,  :default => nil
-    config_param :key_path,     :string,  :default => nil
-    config_param :score_path,   :string,  :default => nil
-    config_param :value_path,   :string,  :default => ''
-    config_param :key_expire,   :integer, :default => -1
-    config_param :value_expire, :integer, :default => -1
-    config_param :value_length, :integer, :default => -1
-    config_param :order,        :string,  :default => 'asc'
+    config_param :format_type,    :string,  :default => 'json'
+    config_param :store_type,     :string,  :default => 'zset'
+    config_param :key_prefix,     :string,  :default => ''
+    config_param :key_suffix,     :string,  :default => ''
+    config_param :key,            :string,  :default => nil
+    config_param :key_path,       :string,  :default => nil
+    config_param :score_path,     :string,  :default => nil
+    config_param :value_path,     :string,  :default => ''
+    config_param :key_expire,     :integer, :default => -1
+    config_param :value_expire,   :integer, :default => -1
+    config_param :value_length,   :integer, :default => -1
+    config_param :order,          :string,  :default => 'asc'
+    config_param :flush_interval, :integer, :default => 1
```

Comment:
Thank you to create and send here a PR.

Well, there is an issue that this patch will change base class' interface, which violates one of OOP principles.
To refer [BufferredOutput](https://github.com/fluent/fluentd/blob/master/lib/fluent/output.rb#L157), `flush_interval` is defined with type of `time`.
```ruby
config_param :flush_interval, :time, default: 60
```

You can use `config_set_default` as like https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/out_exec_filter.rb#L94 .

Plus, once introduce one of derived config parameters, please consider to mention that user can also configure `BufferredOutput`'s parameters.